internal/tests/pytests/wrappers/gfdl_tracker/test_gfdl_tracker.py

```diff
@@ -0,0 +1,247 @@
+import os
+import pytest
+from unittest import mock
+from metplus.wrappers import gfdl_tracker_wrapper as gf
+
+
+time_fmt = "%Y%m%d%H"
+run_times = ["2023080700", "2023080712", "2023080800"]
+
+# Helper class for string matching
+class MatchSubstring(str):
+    def __eq__(self, other):
+        return self in other
+
+def set_minimum_config_settings(config):
+    # set config variables to prevent command from running and bypass check
+    # if input files actually exist
+    config.set("config", "DO_NOT_RUN_EXE", True)
+    config.set("config", "INPUT_MUST_EXIST", False)
+
+    # set process and time config variables
+    config.set("config", "PROCESS_LIST", "GridStat")
```

Comment:
```suggestion
    config.set("config", "PROCESS_LIST", "GFDLTracker")
```
This needs to be changed so that the correct wrapper is initialized.